dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -240,7 +233,7 @@ private static ConstantNameAndType constantNameAndType(
     return (ConstantNameAndType) constantAtNameAndTypeIndex;
   }
 
-  private static ClassSymbolReference constantToClassReference(
+  private static ClassSymbol constantToClassSymbol(
```

Comment:
maybe constantToClassSymbol --> makeSymbol

Reply:
updated to `makeSymbol`